pygaggle/rerank/transformer.py

```diff
@@ -30,34 +31,78 @@
            'QuestionAnsweringTransformerReranker',
            'SentenceTransformersReranker']
 
+prediction_tokens = {
+        'castorini/monot5-base-msmarco':          ['false', 'true'],
```

Comment:
add the underscores here and remove the automatic addition in `get_prediction_tokens`?